project/build.gradle.kts

```diff
@@ -18,6 +18,7 @@ repositories {
 dependencies    {
     // main dependencies
     implementation("org.springframework.boot:spring-boot-starter-web")
+    implementation("org.springframework.boot:spring-boot-starter-cache")
```

Comment:
Why do we need the spring support? Why not use the caffeine cache directly? What does the spring support adds?

Reply:
The initial purpose was to use annotations like "Cacheable" and "EnableCache" but I ended up removing those as they weren't working as expected, the dependency is not needed anymore.

Reply:
Because caffeine cache manager is defined in "import org.springframework.cache.caffeine.CaffeineCacheManager"

Reply:
We don't need anymore forgot to remove it. Was used for annotation like @Cacheable and @EnableCache that ended up being removed, it was replaced by "com.github.ben-manes.caffeine:caffeine:2.8.5" to use caffeine directly.